### PR TITLE
Closes AgileVentures/MetPlus_tracker#479

### PR DESCRIPTION
**Pull request by lollypop27**
Install Jasmine to test JS for utc-to-local-timezone

### CODE REVIEW

File: spec/javascripts/utc_to_local_spec.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+  // https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+describe('Convert UTC to local Time', function () {
+  beforeEach(function () {
+    loadFixtures('utc_to_local/utc_time.html');
```
Comment (patmbolger) [inline spec/javascripts/utc_to_local_spec.js]: You can make most of the houndci issues go away by including these statements at the top of the file (before the `describe` block:
```
var describe;
var loadFixtures;
var spyOn;
var it;
var moment;
var beforeEach;
var utcToLocal;
var expect;
```

File: spec/javascripts/utc_to_local_spec.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  });
+
+  it('New York', function () {
+    spyOn(moment.tz, 'guess').and.returnValue('America/New_York');
+    utcToLocal.setup();
+    ele = $('.utc_to_local_time')[0];
```
Comment (patmbolger) [inline spec/javascripts/utc_to_local_spec.js]: add `var ` in from of `ele` to make this issue go away.

**Comment by lollypop27** [comment]
@patmbolger will hound ever be happy dog?

**Comment by joaopapereira** [comment]
Please rebase this branch

**Comment by lollypop27** [comment]
@joaopapereira and @patmbolger I have rebased and think ready to merge

**Pull request merged**